app/src/main/java/com/mobile/tandil/javabaseproject/entity/Reservation.java

```diff
@@ -0,0 +1,29 @@
+package com.mobile.tandil.javabaseproject.entity;
+
+import com.mobile.tandil.javabaseproject.util.DateUtils;
+import java.util.Calendar;
+
+public class Reservation {
+
+    private final Calendar startingDate;
+    private final Calendar endingDate;
+    private final String key;
+
+    public Reservation(String startingDate, String endingDate, String key) {
```

Comment:
Please, add the parking lot value here. You need all the information of the reservation in this entity

Reply:
done